### PR TITLE
MAGECLOUD-4047: Eliminate PhpExtension class

### PR DESCRIPTION
**Pull request by shiftedreality**
### Fixed Issues (if relevant)

1. https://magento2.atlassian.net/browse/MAGECLOUD-4047

### Manual testing scenarios

1. https://jira.corp.magento.com/browse/MAGECLOUD-126

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

File: src/Command/BuildDist.php [hunk 1]
```diff
@@ -54,5 +54,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         $this->distGenerator->generate();
 
         $output->writeln('<info>Dist files generated</info>');
+
+        return 0;
```
Comment (oshmyheliuk) [inline src/Command/BuildDist.php]: This row is redundant as `Symfony\Console\Command` returns 0 if the command was executed without exceptions.

**Comment by oshmyheliuk** [comment]
@shiftedreality File `php-extensions.php` isn't removed in this PR. Did you forget to remove it, or it still usable?

File: src/Command/BuildDist.php (hunk 1 above)
Comment (shiftedreality) [reply src/Command/BuildDist.php]: Fixed

**Comment by shiftedreality** [comment]
@oshmyheliuk good catch - removed

**Review by oshmyheliuk — APPROVED** [review]

**Review by billygilbert — APPROVED** [review]

**Comment by andriyShevtsov** [comment]
QA approved

**Pull request merged**